src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java

```diff
@@ -115,20 +115,18 @@ public void sessionCreated(IoSession session) throws Exception {
 
         session.getFilterChain().addFirst("proxy", new ProxyFilter(isClient ? "client" : "server"));
 
-        if (true) {
+        String fileName = System.currentTimeMillis() + '_' + forward.getHostName() + '_' + forward.getPort() + '_' + (isClient ? "DOWNSTREAM" : "UPSTREAM");
 
-            String fileName = System.currentTimeMillis() + '_' + forward.getHostName() + '_' + forward.getPort() + '_' + (isClient ? "DOWNSTREAM" : "UPSTREAM");
+        File headersFile = loader.getResource(dumpTo + fileName + ".cap").getFile();
+        headersFile.createNewFile();
 
-            File headersFile = loader.getResource(dumpTo + fileName + ".cap").getFile();
-            headersFile.createNewFile();
+        File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
+        rawFile.createNewFile();
 
-            File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
-            rawFile.createNewFile();
-
-            FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream rawFos = new FileOutputStream(rawFile);
+        try {

```

Comment:
Actually red5 is being built using 1.7 and Paul was going to change it to 1.8 .....
1.6 was set for some compatibility reasons ....